dnf-behave-tests/features/module/remove.feature

```diff
@@ -166,3 +166,43 @@ Given I use repository "dnf-ci-fifthparty"
         | remove                    | leia-0:1.0-1.x86_64    |
         | remove                    | luke-0:1.0-1.x86_64    |
 
+
+Scenario: Remove command respects --all switch to remove all packages
+  Given I successfully execute dnf with args "install nodejs-docs"
+   When I execute dnf with args "module remove nodejs --all"
```

Comment:
What about to add step `"module remove nodejs"` before step with all. The new step should not produce any transaction.

Reply:
Actually it would remove a lot (everything except the nodejs-docs package) - the `default` profile is installed in Background section using `module install nodejs/default`.

Reply:
Sorry, the note was not relevant.